core/scripts/vrfv2plus/testnet/util.go

```diff
@@ -193,7 +193,7 @@ func wrapperConfigure(
 	wrapper, err := vrfv2plus_wrapper.NewVRFV2PlusWrapper(wrapperAddress, e.Ec)
 	helpers.PanicErr(err)
 
-	tx, err := wrapper.SetConfig(
+	tx, err := wrapper.SetConfig0(
```

Comment:
is this a typo?

Reply:
No, I had got this error:
```
$ go run ./vrfv2plus/testnet latest-block-header
# github.com/smartcontractkit/chainlink/core/scripts/vrfv2plus/testnet
vrfv2plus/testnet/util.go:200:3: too many arguments in call to wrapper.SetConfig
        have (*"github.com/ethereum/go-ethereum/accounts/abi/bind".TransactOpts, uint32, uint32, uint8, "github.com/ethereum/go-ethereum/common".Hash, uint8)                                                                 want (*"github.com/ethereum/go-ethereum/accounts/abi/bind".TransactOpts, "github.com/ethereum/go-ethereum/common".Address, uint64)
```

Which is why changed SetConfig to SetConfig0
It was probably changed here: [https://github.com/smartcontractkit/chainlink/commit/7eca961b3b81d9825b267056a5ef1151b95f4e41#diff-82633b19cdf9032b8a86[…]58edf9d5a612a621135844e25d](https://github.com/smartcontractkit/chainlink/commit/7eca961b3b81d9825b267056a5ef1151b95f4e41#diff-82633b19cdf9032b8a86e109dfdf731601f6f558edf9d5a612a621135844e25d)